bin/IdamHelper.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+const fetch = require('node-fetch');
+const { URLSearchParams } = require('url');
+const btoa = require('btoa');
+
+const USERNAME = 'docassemblytestuser@hmcts.net';
+const PASSWORD = '4590fgvhbfgbDdffm3lk4j';
+const FIRSTNAME = 'docassembly';
+const LASTNAME = 'testuser';
+
+const Env = require('./Env');
+
+class IdamHelper {
+
+  constructor() {
+    this.idamUrl = Env.getIdamUrl();
+    this.client = Env.getOAuthClient();
+    this.secret = Env.getOAuthSecret();
+    this.redirect = Env.getOAuthRedirect();
+  }
+
+  async getIdamToken() {
```

Comment:
I think it's normally good practice to add the access modifier and return type:

```
public async getIdamToken(): Promise<string> {
```

Reply:
that is not supported with current version of node/javascript

Reply:
sorry I assumed we were using TypeScript!! We are not... but we should be :)